include/bluetooth/l2cap.h

```diff
@@ -496,6 +507,10 @@ int bt_l2cap_chan_send(struct bt_l2cap_chan *chan, struct net_buf *buf);
 int bt_l2cap_chan_recv_complete(struct bt_l2cap_chan *chan,
 				struct net_buf *buf);
 
+
+#if defined(CONFIG_BT_TESTING)
+int bt_l2cap_chan_reconfigure(struct bt_l2cap_chan **chan, uint16_t mtu);
```

Comment:
Missing documentation (even if it just a testing function). 

Also 
```suggestion
int bt_l2cap_chan_reconfigure(struct bt_l2cap_chan **chans, uint16_t mtu);
```

Reply:
again, I named it 'chan' to mimic bt_l2cap_ecred_chan_connect()

Reply:
`bt_l2cap_ecred_chan_connect` uses `chans`:

```
int bt_l2cap_ecred_chan_connect(struct bt_conn *conn,
				struct bt_l2cap_chan **chans, uint16_t psm);
```

Reply:
hmm looks like there is discrepancy between definition and declaration of that function